libsolidity/formal/SMTEncoder.cpp

```diff
@@ -2688,7 +2688,28 @@ vector<VariableDeclaration const*> SMTEncoder::stateVariablesIncludingInheritedA
 
 vector<VariableDeclaration const*> SMTEncoder::localVariablesIncludingModifiers(FunctionDefinition const& _function, ContractDefinition const* _contract)
 {
-	return _function.localVariables() + modifiersVariables(_function, _contract);
+	return _function.localVariables() + tryCatchVariables(_function) + modifiersVariables(_function, _contract);
+}
+
+vector<VariableDeclaration const*> SMTEncoder::tryCatchVariables(FunctionDefinition const& _function)
+{
+	struct TryCatchVarsVisitor : public ASTConstVisitor{
+		vector<VariableDeclaration const*> m_vars;
+
+		bool visit(TryCatchClause const& _catchClause) override
+		{
+			if (_catchClause.parameters())
+			{
+				auto const& params = _catchClause.parameters()->parameters();
+				for (auto param: params)
+					m_vars.push_back(param.get());
+			}
```

Comment:
Are the return parameters also included here?

Reply:
If by return parameters you mean the `x` and `b` in `try d.d() returns (uint x, bool b) ...`, then yes. They are stored as the parameters of the success clause in the AST representation.

Reply:
Ok!